lang/de.js

```diff
@@ -16,7 +16,7 @@ export default {
 	"components.form-element.input.text.tooShort": "{label} muss mindestens {minlength} Zeichen enthalten",
 	"components.form-element.input.url.typeMismatch": "URL ist ungültig",
 	"components.form-element.valueMissing": "{label} ist erforderlich.",
-	"components.form-error-summary.errorSummary": "In den von Ihnen übermittelten Informationen {count, plural, one { ist 1 Fehler } other { sind { count } Fehler }} enthalten",
```

Comment:
Do these spaces that were removed matter?

Reply:
I think it's better without the spaces since they'll be part of the value that gets inserted, resulting in double spaces.